prepare_sdk.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Please move the script to the rnsdk folder and run it within.

Reply:
Done